composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "cakephp/cakephp": "~3.0"
+        "cakephp/cakephp": "dev-master"
```

Comment:
I'll probably do 3.0.7 in a week or so. We'll have to not release bake until after that :smile: